src/index.ts

```diff
@@ -116,7 +118,11 @@ const App = async () => {
       clearStatus()
       showStatus(`<pre>${JSON.stringify(data, null, 2)}</pre>`, COLORS.success)
     } catch (err) {
-      showStatus(`<pre>${err.stack ?? err.message}</pre>`, COLORS.error)
+      if (err instanceof Error) {
+        showStatus(`<pre>${err.stack ?? err.message}</pre>`, COLORS.error)
+      } else {
+        showStatus(`<pre>${err}</pre>`, COLORS.error)
+      }
```

Comment:
:+1:

we might want to actually `console.error` the error if it doesn't have `.stack`/`.message` or isn't `instanceof Error`